config/crds/hive_v1_machinepool.yaml

```diff
@@ -0,0 +1,178 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: machinepools.hive.openshift.io
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .spec.name
+    name: PoolName
+    type: string
+  - JSONPath: .spec.clusterDeploymentRef.name
+    name: ClusterDeployment
+    type: string
+  - JSONPath: .spec.replicas
+    name: Replicas
+    type: integer
+  group: hive.openshift.io
+  names:
+    kind: MachinePool
+    plural: machinepools
```

Comment:
Can we add a shortname `mp`.

Reply:
If someone feels strongly about adding a short name, then it can be added in a subsequent PR. I personally do not like proliferating short names.